### ISSUE TITLE
Luckperms group with "*" permission

### ISSUE THREAD

**Issue by Asherslab** [Zerthick/PlayerShopsRPG#32]
Hey, so i've got an Admin rank using "Luckperms" (on a sponge server 1.10.2) and the rank has the "*" perms, my admins can't use any of the /shop commands unless  i explicitly give them the permissions for each command (Not that hard, but i doubt that's the intended use) i also tried 
- playershopsrpg.*
- playershopsrpg.commands.*
but neither worked to give the commands

`PlayerShopsRPG version: 2.0.1 by Zerthick`

**Comment by Zerthick** [comment]
I don't believe luckperms uses the `*` permission. Have you tried giving your admins `playershopsrpg.commands`?

**Comment by Asherslab** [comment]
It does use the "*" permission, as my admins have that permission, and they have access to all of the other server commands (no inheritance either so it's definitely the *) the only one that doesn't seem to work is the playershopsrpg on, I also tried playershopsrpg.* and playershopsrpg.commands.*

**Comment by Zerthick** [comment]
Any luck? Or is it still not working?

**Comment by Asherslab** [comment]
Nope, works fine. Thanks
